scenes/SceneProfile.js

```diff
@@ -98,62 +91,15 @@ export default class SceneProfile extends React.Component {
       );
     }
 
-    return <ProfilePage {...this.props} data={this.state.profile} />;
-  }
-}
-
-class ProfilePage extends React.Component {
-  state = {
-    isFollowing: !!this.props.viewer.subscriptions.filter((entry) => {
-      return entry.target_user_id === this.props.data.id;
-    }).length,
-  };
-
-  componentDidUpdate = (prevProps) => {
-    if (
-      this.props.data.id !== prevProps.data.id ||
-      this.props.viewer.subscriptions !== prevProps.viewer.subscriptions
-    ) {
-      this.setState({
-        isFollowing: !!this.props.viewer.subscriptions.filter((entry) => {
-          return entry.target_user_id === this.props.data.id;
-        }).length,
-      });
-    }
-  };
-
-  _handleFollow = async () => {
-    this.setState({ isFollowing: !this.state.isFollowing });
-    await Actions.createSubscription({
-      userId: this.props.data.id,
-    });
-  };
-
-  render() {
-    let buttons = (
-      <div css={STYLES_BUTTONS}>
-        {this.state.isFollowing ? (
-          <ButtonSecondary style={{ marginRight: 8 }} onClick={this._handleFollow}>
-            Unfollow
-          </ButtonSecondary>
-        ) : (
-          <ButtonPrimary style={{ marginRight: 8 }} onClick={this._handleFollow}>
-            Follow
-          </ButtonPrimary>
-        )}
-      </div>
-    );
+    //FOR TARA: this is the change that'll allow it to update "automatically" if you're viewing your own profile and make changes
```

Comment:
Please stick to our comment convention

`NOTE(Tara):`

Reply:
@jimmylee these were meant to be temporary comments I left for tara so she knows what I changed and why. Meant for them to be easily found by CTRL + F and deleted after so I used a different format

Reply:
@tarafanlin think you missed this one. I'll take it out. This one is important to know since this was the cause of the stale data issues you were having. Was just a small change that made all the difference

Reply:
oh yes sorry I only CTRL+F inside of profile.js, forgot about this one. I will review the comment and take it out!